src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length < 1) {
+    return {};
+  }
+
+  const sourceElements = sourceString.split(';');
+
+  const sourceObject = sourceElements
+    .map((el) => el.trim())
+    .filter((el) => el.length > 0)
+    .reduce((obj, el) => {
+      const key = el.slice(0, el.indexOf(':')).trim();
+      const value = `${el.slice(el.indexOf(':') + 1, el.length).trim()}`;
```

Comment:
You can directly return the trimmed value without using template literals since no additional string manipulation or interpolation is being done.